components/CreateSpot.js

```diff
@@ -0,0 +1,214 @@
+        <select id="category" name="category" required>
+          <option value="">--Bitte eine Kategorie auswählen--</option>
+          <option value="Spielplatz">Spielplatz</option>
+          <option value="Indoor Spielplatz">Indoor Spielplatz</option>
+          <option value="Spielen">Spielen</option>
+          <option value="Sport">Sport</option>
+          <option value="Sportplatz">Sportplatz</option>
+          <option value="Sporthalle">Sporthalle</option>
+          <option value="Sportkurs">Sportkurs</option>
+          <option value="Klettern">Klettern</option>
+          <option value="Skaten">Skaten</option>
+          <option value="Wintersport">Wintersport</option>
+          <option value="Schwimmbad">Schwimmbad</option>
+          <option value="Freibad">Freibad</option>
+          <option value="Badesee">Badesee</option>
+          <option value="Strand & Meer">Strand & Meer</option>
+          <option value="Park">Park</option>
+          <option value="Wald">Wald</option>
+          <option value="Kultur">Kultur</option>
+          <option value="Musik">Musik</option>
+          <option value="Kunst">Kunst</option>
+          <option value="Zirkus">Zirkus</option>
+          <option value="Basteln">Basteln</option>
+          <option value="Essen & Trinken">Essen & Trinken</option>
+          <option value="Übernachtung">Übernachtung</option>
+          <option value="Freizeitpark">Freizeitpark</option>
+          <option value="Jahrmarkt">Jahrmarkt</option>
+          <option value="Zoo">Zoo</option>
+          <option value="Tiere">Wildpark</option>
+          <option value="Bauernhof">Bauernhof</option>
+          <option value="Ponyreiten">Ponyreiten</option>
+          <option value="ParentHood Spot">Sonstiger Spot</option>
+        </select>
+        <FormLabels for="name">Name*</FormLabels>
+        <input id="name" name="name" type="text" required />
+        <FormLabels for="addresse">Adresse*</FormLabels>
+        <input id="addresse" name="addresse" type="text" required />
+        <Checkbox>
+          <Legend>Wetterempfehlung</Legend>
+          <div>
+            <input
+              id="good_weather"
+              name="good_weather"
+              type="checkbox"
+              value="bei gutem Wetter"
+            />
+            <label for="weathersuggestion">bei gutem Wetter</label>
+          </div>
+          <div>
+            <input
+              id="moderate_weather"
+              name="moderate_weather"
+              type="checkbox"
+              value="bei mäßigem Wetter"
+            />
+            <label for="weathersuggestion">bei mäßigem Wetter</label>
+          </div>
+          <div>
+            <input
+              id="bad_weather"
+              name="bad_weather"
+              type="checkbox"
+              value="bei schlechtem Wetter"
+            />
+            <label for="weathersuggestion">bei schlechtem Wetter</label>
+          </div>
```

Comment:
☝️ A small rule of thumb is: if you write the same code three times, it's time to outsource it as a reusable component - but it's up to you.
Anyway the small `htmlFor - attribute` bug is everywhere 🪲


```suggestion
            <input
              id="good_weather"
              name="good_weather"
              type="checkbox"
              value="bei gutem Wetter"
            />
            <label htmlFor="weathersuggestion">bei gutem Wetter</label>
            <input
              id="moderate_weather"
              name="moderate_weather"
              type="checkbox"
              value="bei mäßigem Wetter"
            />
            <label htmlFor="weathersuggestion">bei mäßigem Wetter</label>
            <input
              id="bad_weather"
              name="bad_weather"
              type="checkbox"
              value="bei schlechtem Wetter"
            />
            <label htmlFor="weathersuggestion">bei schlechtem Wetter</label>
   
```